plugins/woocommerce/src/Internal/RestApi/Routes/V4/Customers/CollectionQuery.php

```
@@ -181,29 +166,22 @@ public function get_query_args( WP_REST_Request $request ): array {
 	 * @return array
 	 */
 	public function get_query_results( array $query_args, WP_REST_Request $request ): array {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)